### PR TITLE
feat(developers): include absolute view path in view wrapper

### PR DESCRIPTION
**Pull request by hypeJunction**
Make it easier to locate the view by adding full view location to the view wrapper

### CODE REVIEW

**Review by jdalsem — APPROVED** [review]
Not using this ... ever... but change LGTM

**Comment by hypeJunction** [comment]
It's quite helpful when you are looking for a plugin that hijacks the groups edit form, check all obvious offenders at no avail, to discover it's profile manager :)

**Comment by mrclay** [comment]
Would rather have a URL to the entry in the Inspect view page, which includes a lot more info.

File: mod/developers/start.php [hunk 1]
```diff
@@ -190,7 +191,8 @@ function developers_wrap_views($hook, $type, $result, $params) {
 	}
 
 	if ($result) {
-		$result = "<!-- developers:begin $view -->$result<!-- developers:end $view -->";
+		$filename = _elgg_services()->views->findViewFile($view, 'default');
```
Comment (hypeJunction) [inline mod/developers/start.php]: In that case it should also include all hooks that are modifying the view. I just want something simple that tells me which plugin is messing with my presentation.

**Comment by hypeJunction** [comment]
This is the best I can do. The url is useless in the html comment - it will take you more time to copy the url, then to keep a separate tab open and find it with search. If you don't like it, let's just drop this.

inspector

**Comment by hypeJunction** [comment]
Spent too much time on this already. Abandoning.

**Comment by jdalsem** [comment]
I liked the initial solution (with just the path to the providing view file...) better than the second one... Reopen if you are willing to change to that approach... then we can merge it

**Pull request closed without merging**